community/detectors/apache_apisix_cve_2020_13945/src/main/java/com/google/tsunami/plugins/detectors/rce/apache_default_token/ApacheDefaultTokenDetector.java

```diff
@@ -0,0 +1,210 @@
+ * limitations under the License.
+ */
+package com.google.tsunami.plugins.detectors.rce.apache_default_token;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.net.HttpHeaders.CONTENT_TYPE;
+import static com.google.tsunami.common.data.NetworkEndpointUtils.toUriAuthority;
+import static com.google.tsunami.common.net.http.HttpRequest.get;
+import static com.google.tsunami.common.net.http.HttpRequest.post;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import com.google.common.flogger.GoogleLogger;
+import com.google.common.net.MediaType;
+import com.google.inject.Inject;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.util.Timestamps;
+import com.google.tsunami.common.data.NetworkServiceUtils;
+import com.google.tsunami.common.net.http.HttpClient;
+import com.google.tsunami.common.net.http.HttpHeaders;
+import com.google.tsunami.common.net.http.HttpResponse;
+import com.google.tsunami.common.time.UtcClock;
+import com.google.tsunami.plugin.PluginType;
+import com.google.tsunami.plugin.VulnDetector;
+import com.google.tsunami.plugin.annotations.PluginInfo;
+import com.google.tsunami.proto.DetectionReport;
+import com.google.tsunami.proto.DetectionReportList;
+import com.google.tsunami.proto.DetectionStatus;
+import com.google.tsunami.proto.NetworkService;
+import com.google.tsunami.proto.Severity;
+import com.google.tsunami.proto.TargetInfo;
+import com.google.tsunami.proto.Vulnerability;
+import com.google.tsunami.proto.VulnerabilityId;
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.time.Clock;
+import java.time.Instant;
+
+/** A {@link VulnDetector} that detects Apache APISIX RCE CVE-2020-13945. */
+@PluginInfo(
+    type = PluginType.VULN_DETECTION,
+    name = "Apache APISIX with default Admin token Detector",
+    version = "0.1",
+    description = "This detector checks Apache APISIX with default Admin token.",
+    author = "hh-hunter",
+    bootstrapModule = ApacheDefaultTokenDetectorBootstrapModule.class)
+public final class ApacheDefaultTokenDetector implements VulnDetector {
+  private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
+
+  @VisibleForTesting static final String DETECTION_STRING = "tsunami_verify_success";
+
+  @VisibleForTesting
+  static final String VULN_DESCRIPTION =
+      "Apache APISIX 1.2, 1.3, 1.4, and 1.5 is susceptible to insufficiently protected credentials. An attacker can "
+          + "enable the Admin API and delete the Admin API access IP restriction rules. Eventually, the default token "
+          + "is allowed to access APISIX management data. This vulnerability allows an attacker to execute arbitrary "
+          + "code on the server.";
```

Comment:
The vulnerability description is still the one from the CVE, please update the description to reflect the "default admin token" bug.